ci/cron/daily-compat.yml

```diff
@@ -170,7 +170,7 @@ jobs:
           BAZEL_DEPENDENCY_TYPE="haskell_cabal_library"
 
           bash <(curl -s https://raw.githubusercontent.com/DACH-NY/security-blackduck/master/synopsys-detect) \
-          ci-build digital-asset_daml $(Build.SourceBranchName) \
+          ci-build digital-asset_daml local-main-maven \
```

Comment:
@garyverhaegen-da i think we want to keep this as the real branch name rather than hardcode

Reply:
changed it to main -- (this should basically be equivalent to the current state but let's see if it helps)

Reply:
What if we're not running this on main?